.github/workflows/build.yml

```diff
@@ -123,14 +123,24 @@ jobs:
 
             - name: Configure and build.
               run : |
-                  cmake -S .    --preset=${{ matrix.compiler_family }}-${{ matrix.backend }} --warn-uninitialized -Werror=dev
+                  cmake -S .    --preset=${{ matrix.compiler_family }}-${{ matrix.backend }} --warn-uninitialized -Werror=dev -DCMAKE_INSTALL_PREFIX=/opt/test-folder-for-install
                   cmake --build --preset=${{ matrix.compiler_family }}-${{ matrix.backend }}
 
+            - name: Install.
+              run : |
+                  bash tests/install.sh ${{ matrix.compiler_family }}-${{ matrix.backend }} /opt/test-folder-for-install
```

Comment:
Let's move it to the step after `Test` to make it "grouped" with the last step?

Reply:
The two could also become a single step.

Reply:
Kept it 2 steps.